### PR TITLE
[APP-2415] Refactor aptoide games settings

### PR DESCRIPTION
**Pull request by pedroagostinhoaptoide**
**What does this PR do?**

   - Adds the specific version name and version code to the settings screen.
   - Removes UI theme options from the settings screen.
   - Refactors the legal section in the settings: removes the opt in/opt out toggle and adds Privacy Policy and Terms & Condition redirect buttons.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] See by commit.

**How should this be manually tested?**

  Flow on how to test this or QA Tickets related to this use-case: [APP-2415](https://aptoide.atlassian.net/browse/APP-2415)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-2415](https://aptoide.atlassian.net/browse/APP-2415)

**Code Review Checklist**

- [ ] Documentation on public interfaces
- [ ] Database changed? If yes - Migration?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] New Kotlin code has unit tests
- [ ] New flows in presenters unit tests
- [ ] Mappers/Validators with any kind of logic unit tests
- [ ] Functional tests pass

[APP-2415]: https://aptoide.atlassian.net/browse/APP-2415?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[APP-2415]: https://aptoide.atlassian.net/browse/APP-2415?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app-games/src/main/java/com/aptoide/android/aptoidegames/settings/SettingsScreen.kt [hunk 1]
```diff
@@ -76,6 +77,8 @@ fun NavGraphBuilder.settingsScreen(
     downloadOnlyOverWifi = downloadOnlyOverWifi,
     isDarkTheme = isDarkTheme,
     acceptedPPAndTC = acceptedPPAndTC,
+    verName = BuildConfig.VERSION_NAME,
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/settings/SettingsScreen.kt:77]: It is the same value as current default value in the `SettingsViewContent` composable 🤔

File: app-games/src/main/java/com/aptoide/android/aptoidegames/settings/SettingsScreen.kt [hunk 2]
```diff
@@ -76,6 +77,8 @@ fun NavGraphBuilder.settingsScreen(
...
+    verCode = BuildConfig.VERSION_CODE,
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/settings/SettingsScreen.kt:78]: It is the same value as current default value in the `SettingsViewContent` composable 🤔

File: app-games/src/main/res/values/strings.xml
Comment (G0retZ) [inline app-games/src/main/res/values/strings.xml:1]: Lets not touch string resources to not interfer with @carlosgarridoaptoide changes

**Review by G0retZ — CHANGES_REQUESTED** [review]

File: app-games/src/main/res/values/strings.xml
Comment (pedroagostinhoaptoide) [reply app-games/src/main/res/values/strings.xml:1]: The app-games project is still not tracked by crowding, so we've been changing the string file directly

File: app-games/src/main/java/com/aptoide/android/aptoidegames/settings/SettingsScreen.kt (hunk 2 above)
Comment (pedroagostinhoaptoide) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/settings/SettingsScreen.kt:78]: The `SettingsViewContent` composable has the default values "1.2.3" and "123" which should only be used for previews, hence this change. However, app-games should probably have its own versioning system using Gradle properties

File: app-games/src/main/java/com/aptoide/android/aptoidegames/settings/SettingsScreen.kt (hunk 1 above)
Comment (pedroagostinhoaptoide) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/settings/SettingsScreen.kt:77]: The `SettingsViewContent` composable has the default values "1.2.3" and "123" which should only be used for previews, hence this change. However, app-games should probably have its own versioning system using Gradle properties

**Review by G0retZ — APPROVED** [review]

**Pull request merged**